test/grizzly/connection_test.exs

```diff
@@ -4,7 +4,7 @@ defmodule Grizzly.ConnectionTest do
   alias Grizzly.{Connection, Report}
 
   test "open a connection" do
-    assert {:ok, _connection} = Connection.open(1)
+    assert {:ok, _connection} = Connection.open(1200)
```

Comment:
Is this an intended change?

Reply:
yeah, the tests were failing because we already had connected to node id 1.

Honestly, we can probably get rid of this test because so many other areas of the code call this line that other tests would fail if we changed the API or broke something here.